tests/test_auditing.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 DB Netz AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import capellambse
+from capellambse import auditing
+
+
+def test_AttributeAuditor_tracks_uuids_of_accessed_ModelElements(
+    model: capellambse.MelodyModel,
+):
+    # pylint: disable=pointless-statement
+    comp, comp1, comp2 = model.la.all_components[:3]
+    uuids = {comp.uuid, comp2.uuid}
```

Comment:
A better name here IMO would be `expected_uuids` (or just `expected`, as is used in various other tests).

Also, for clarity, an empty line after this one would be good, to signify the end of the "setup phase" for this test.